app/src/main/res/values/arrays.xml

```diff
@@ -424,6 +424,17 @@
     </string-array>
     <string name="webdav_scheme_default" translatable="false">HTTP</string>
 
+    <!-- Immich schemes -->
```

Comment:
We could rename webdav_scheme_entries to something like common_scheme_entries and use it for WebDAV and Immich?